Library/Homebrew/cmd/configure-shell.rb

```diff
@@ -0,0 +1,19 @@
+module Homebrew
+  module_function
+
+  def configure_shell
+    # Write environment details (PATH, MANPATH, and INFOPATH) to a file that will export them
+    environment = <<~EOS
+      # Homebrew environment
+      # Created by running brew configure-shell
+      export PATH="#{HOMEBREW_PREFIX}/bin:$PATH"
+      export MANPATH="#{HOMEBREW_PREFIX}/share/man:$MANPATH"
+      export INFOPATH="#{HOMEBREW_PREFIX}/share/info:$INFOPATH"
+    EOS
+    File.write HOMEBREW_PREFIX/"etc/environment", environment, perm: 0755
```

Comment:
I forgot that a shell snippet doesn't need to have execute permissions to be able to `source` it. Please remove `, perm: 0755`.